rushScripts/regeneration.js

```diff
@@ -48,7 +48,7 @@ for (namespace in goMappings) {
 } 
 
 const blobStorage = 'https://raw.githubusercontent.com/Azure/azure-rest-api-specs/storage-dataplane-preview/specification/storage/data-plane/Microsoft.BlobStorage/preview/2019-07-07/blob.json';
-generate(blobStorage, 'test/storage/2019-07-07/azblob', '--credential-scope="https://storage.azure.com/.default" --module="azstorage"');
+generate(blobStorage, 'test/storage/2019-07-07/azblob', '--credential-scope="https://storage.azure.com/.default" --module="azstorage" --export-client="false"');
```

Comment:
Why dont we want to export clients for storage?

Reply:
This is just for testing purposes.  For the actual shipping bits, the storage team wants to export hand-written clients created in the convenience layer that consume the internally generated clients (this is what they did in track 1.8).

Reply:
Got it. Thanks for the info!